API Project/src/main/java/hello/User.java

```diff
@@ -3,13 +3,74 @@
 public class User {
   private String firstName;
   private String lastName;
+  private String email;
+  private String userName;
+  private String password;
+  //private long id?
```

Comment:
We need the `id` field, just in case we might want to use a database in the future. Id may not have a setter for now